src/components/useThree/mainCanvas.tsx

```diff
@@ -16,19 +17,36 @@ export const MainCanvas = (props: Props) => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const { moonquakeData, setIsMap, option, choiceMoonquake, setChoiceMoonquake } = props;
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const orbitControlsRef = useRef<any>(null);
+
+  const handleOrbitControlsChange = () => {
+    const orbitControls = orbitControlsRef.current;
+    if (!orbitControls) return;
+    if (orbitControls.getDistance() === orbitControls.minDistance) {
```

Comment:
```suggestion
    if (orbitControls.getDistance() <= orbitControls.minDistance) {
```

じゃないと、早く動かした時や一部のデバイスで変わってくれなさそう